buffer/src/main/java/io/netty/buffer/AbstractByteBufAllocator.java

```diff
@@ -34,7 +34,7 @@
         ResourceLeakDetector.addExclusions(AbstractByteBufAllocator.class, "toLeakAwareBuffer");
     }
 
-    protected static ByteBuf toLeakAwareBuffer(ByteBuf buf) {
+    protected ByteBuf toLeakAwareBuffer(ByteBuf buf) {
```

Comment:
This is a breaking change so you will need to preserve the old method signature as well as people may depend on it to be static atm when sub-classing `AbstractByteBufAllocator`